tests/v2/validation/rbac/rbac.go

```diff
@@ -10,15 +14,30 @@ import (
 	"github.com/rancher/rancher/tests/framework/extensions/projects"
 	"github.com/rancher/rancher/tests/framework/extensions/users"
 	password "github.com/rancher/rancher/tests/framework/extensions/users/passwordgenerator"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads"
 	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	appv1 "k8s.io/api/apps/v1"
+	coreV1 "k8s.io/api/core/v1"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
-const roleOwner = "cluster-owner"
-const roleMember = "cluster-member"
-const roleProjectOwner = "project-owner"
-const roleProjectMember = "project-member"
+const (
+	roleOwner = "cluster-owner"
+	roleMember = "cluster-member"
+	roleProjectOwner = "project-owner"
+	roleProjectMember = "project-member"
+	roleProjectReadOnly = "read-only"
+	restrictedAdmin = "restricted-admin"
+	standardUser = "user"
+	pssRestrictedPolicy = "restricted"
+	pssBaselinePolicy = "baseline"
+	pssPrivilegedPolicy = "privileged"
+	psaWarn = "pod-security.kubernetes.io/warn"
+	psaAudit = "pod-security.kubernetes.io/audit"
+	psaEnforce = "pod-security.kubernetes.io/enforce"
+)
```

Comment:
Just to make sure, this is also gonna change with PSA PR as we've talked, adding this note as a reminder for us that this PR will need a rebase.